### DISCUSSION TITLE
Can you interactively edit an existing manifest?

### DISCUSSION THREAD

**Discussion by marco-eckstein** [microsoft/winget-create#496] [Q&A] [answered]
With the `New` command, you can edit the manifest interactively, which helps with adhering to the schema. But I have yet to find a way to edit an existing manifest interactively. Is there one?

**Comment by mdanish-kh** [comment, marked as answer]
You can use the `wingetcreate update` command with the `--interactive` switch that lets you edit an existing manifest.

Alternatively, you can clone https://github.com/microsoft/winget-pkgs locally, install the Red Hat YAML extension in VSCode and edit an existing manifest. If the manifest contains a schema header, the extension will help in offering intellisense that conforms to the schema.

Finally, after making any change to an existing manifest, you can always use `winget validate -m <path_to_manifest>` to see if your changes conform to the schema

**Reply by marco-eckstein** [reply]
Thanks -I see. `--interactive` is missing from the [docs](https://github.com/microsoft/winget-create/blob/main/doc/update.md), which even state that _the update command is non-interactive_. It is included in `--help` though.

Which header is recommended when manually editing a manifest that contains a header like this?

```yaml
# Created using wingetcreate 1.5.7.0
# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
```

Should the comment about wingetcreate be removed?

**Reply by mdanish-kh** [reply]
Ah, I see. The docs need to be updated, I just haven't found the time to go through the code and update the examples in `wingetcreate update --help` and also the docs `.md` pages.

Regarding the comment, it doesn't really matter. You can remove the comment if you feel like. Just don't remove the `# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json` line as it helps people using the YAML extension get intellisense suggestions
